frontend/src/components/templates/InstructorEnrollmentList.tsx

```diff
@@ -94,6 +95,45 @@ const InstructorEnrollmentList: React.FC = () => {
     };
   }, [pagination, sorted, filtered]);
 
+  const targetApiColumn = columnHelper.accessor(
+    ({ target_api }: { target_api: string }) =>
+      dataProviderConfigurations?.[target_api].label,
+    {
+      header: 'Fournisseur',
+      id: 'target_api',
+      enableSorting: false,
+      meta: {
+        filter: 'select',
+        selectOptions: user?.roles
+          .filter((role) => role.endsWith(':reporter'))
+          .map((role) => {
+            const targetApiKey = role.split(':')[0];
+
+            return {
+              key: targetApiKey,
+              label: dataProviderConfigurations?.[targetApiKey].label,
+            };
+          }),
+      },
+      filterFn: 'arrIncludesSome',
+    }
+  );
+
+  const zipCodeColumn = columnHelper.accessor('zip_code', {
+    header: 'Code postal',
+    enableSorting: false,
+    filterFn: 'includesString',
+    meta: {
+      placeholder: 'ex : 75009',
+    },
+  });
+
+  const changingColumn = user?.roles.some((role) =>
+    role.endsWith('hubee_portail_cnaf:reporter')
+  )
+    ? zipCodeColumn
```

Comment:
If we are administrator of all api, we have the column targetApiColumn who just disappears. It does not include much people: mostly : us (devs, Nico and Anais). 
We are sure that's the require behaviour that we want ?

Reply:
As seen with @SchweisguthN, we can agree that yes it's the desired behaviour.